src/app/models/work-item.spec.ts

```diff
@@ -239,7 +239,7 @@ describe('WorkItemMapper', () => {
           'labels': {
             'data': []
           },
-          'system.boardColumns': {},
+          'boardColumns': {},
```

Comment:
@jarifibrahim why do you call this `boardColumns` and not `boardcolumns`?

Reply:
I thought it's in camel case. I've changed it in https://github.com/fabric8-ui/fabric8-planner/pull/2803/commits/3399dfeee30353446153111446696adce91d9d86

Reply:
@kwk why not use camel case?